src/test/git-elegant-clone.bats

```diff
@@ -7,7 +7,13 @@ load fake-cd
 setup() {
     fake-pass git clone
     fake-pass git "clone https://github.com/extsoft/elegant-git.git"
-    fake-pass git "elegant configure --local"
```

Comment:
It's a bad idea to replace `git elegant configure --local` mock with the mocks for everything this command does. Please find another way.